scripts/analysis.py

```diff
@@ -56,188 +25,85 @@ def get_agent_ids(cursor, facility):
     cursor: cursor
         cursor for sqlite3
     facility: str
-        name of facility type
+        name of facility type as described in spec
 
     Returns
     -------
-
     sink_id: list
-        list of all the sink agentId values.
+        list of all the sink agentId values as string.
```

Comment:
1. Strings are returned because for all the functions in the analysis.py script utilizes string functions to make sqlite query strings. So rather than going through the returned list of agentid integers and casting them as strings, we thought it would be better for the function to simply return them as a list of all sink agentid strings.

2. Thank you for pointing that out! It meant that the elements are cast as strings. I will fix this.